sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -37,7 +37,16 @@
   // the following fields are not final to avoid compile-time constants used by plugins
   public static String DOMAIN_SIZE = "Size";
   public static String DOMAIN_TESTS = "Tests";
+
+  /**
+   * @deprecated in 5.5. Merged into {@link #DOMAIN_TESTS}
+   */
+  @Deprecated
   public static String DOMAIN_INTEGRATION_TESTS = "Tests (Integration)";
+  /**
+   * @deprecated in 5.5. Merged into {@link #DOMAIN_TESTS}
+   */
+  @Deprecated
   public static String DOMAIN_OVERALL_TESTS = "Tests (Overall)";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)